stock_indicators/_cslib/__init__.py

```diff
@@ -24,52 +21,16 @@
 
 logger = logging.getLogger(__name__)
 
-
-def verify_dotnet_environment():
-    """Verify .NET environment setup"""
-    dotnet_root = os.environ.get("DOTNET_ROOT")
-    logger.debug("DOTNET_ROOT: %s", dotnet_root)
-
-    if platform.system() == "Darwin":
-        if not dotnet_root:
-            os.environ["DOTNET_ROOT"] = "/usr/local/share/dotnet"
-            logger.debug("Setting DOTNET_ROOT for macOS")
-
-
 try:
-    verify_dotnet_environment()
```

Comment:
no issue with removing this in favor of the built-in `pythonnet` [find.py](https://github.com/pythonnet/clr-loader/blob/master/clr_loader/util/find.py#L19-L59) utility, if it works, but you may have to explicitly call it here.

Reply:
@DaveSkender Built-in utility(`find_dotnet_root()`) is called inside `load()`. we can call explicitly but it might be redundant.